WebApplication/ClientApp/src/components/AuthService.jsx

```diff
@@ -28,3 +28,11 @@ export const logout = () => {
 export const getProfile = () => {
   return decode(getToken()).unique_name;
 };
+
+export const setLanguage = language => {
```

Comment:
Can you move language storing to another component? This is user authentification service so language shouldn't be here

Reply:
good point, fixed that